lib/cooked_post_processor.rb

```diff
@@ -76,8 +77,6 @@ def post_process_images
       limit_size!(img)
       convert_to_link!(img)
     end
-
-    update_post_image
```

Comment:
Is there a reason we need this to be in the mutex? I'm assuming there must be a reason it was placed out of the mutex in the first place.